dev-cpp/htmlcxx/htmlcxx-0.87.recipe

```diff
@@ -38,6 +38,7 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
```

Comment:
Not needed here.

Reply:
Without this line, it doesn't work correctly. It complains about same missing symbol as with all recipes that depend on this recipe.